bigquery_etl/cli/check.py

```diff
@@ -123,24 +123,37 @@ def _run_check(
         **{"dataset_id": dataset_id, "table_name": table},
         **parameters,
     }
+    rendered_result = render_template(
+        checks_file.name,
+        template_folder=str(checks_file.parent),
+        templates_dir="",
+        format=False,
+        **jinja_params,
+    )
+    checks = rendered_result.split(";")
```

Comment:
I recommend using `sqlparse` here instead to determine the individual SQL statements. Similar to how we use it here: https://github.com/mozilla/bigquery-etl/blob/d82acc18562df147c1b2e418498ff2f74067d8d8/bigquery_etl/routine/parse_routine.py#L143

Otherwise we might run into issues where these tests unexpectedly break because `;` are used elsewhere, like in a comment.

Reply:
Oops missed that, updated.